packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.ts

```diff
@@ -28,6 +28,23 @@ export const disableAdd = (
       }
       break;
     }
+    case 'gauge': {
```

Comment:
is this different than the 2 cases above?

Reply:
Sorry, both are same. Resolved